### PR TITLE
#163218155 Increase the assertions in the tests

### PR DESCRIPTION
**Pull request by khwilo**
#### What does this PR do?

Adds more assertions to the tests.

#### Description of Task to be completed?

- [x] Implement more than one assertion type in the tests
- [x] Separate the logic between the tests
- [x] Update the HTTP status codes

#### How should this be manually tested?

1. Clone the repository and cd into it.
2. Create a virtual environment  and activate it:
```bash
$ python3 -m venv venv
$ source venv/bin/activate
```
3. Install the required dependencies:
```bash
$ pip install -r requirements.txt
```
4. Run the tests
```bash
$ pytest --cov=app/
```

#### What are the relevant pivotal tracker stories?

[#163218155](https://www.pivotaltracker.com/story/show/163218155)

#### Screenshot

test_run

### CODE REVIEW

**Review by bencyn — APPROVED** [review]

**Comment by bencyn** [comment]
Your test coverage is appealing. Looks good to me.

**Comment by emdeechege** [comment]
Nice coverage

File: app/tests/v1/test_meetup.py [hunk 1]
```diff
@@ -7,41 +7,32 @@ class MeetupTestCase(BaseTestCase):
...
-        access_token = response_msg["access_token"]
+        access_token = self.get_access_token(self.admin_registration, self.admin_login)
         res = self.client().post(
             '/api/v1/meetups',
             headers=self.get_authentication_headers(access_token),
             data=json.dumps(self.meetup)
         )
+        response_msg = json.loads(res.data.decode("UTF-8"))
         self.assertEqual(res.status_code, 201)
+        self.assertTrue(response_msg['data'])
```
Comment (RuthNjeri) [inline app/tests/v1/test_meetup.py:19]: Assert the content of the data as well.

File: app/tests/v1/test_meetup.py [hunk 2]
```diff
@@ -51,17 +42,10 @@ def test_fetch_one_meetup(self):
         self.assertEqual(response_msg["status"], 200)
         self.assertTrue(response_msg["data"])
```
Comment (RuthNjeri) [inline app/tests/v1/test_meetup.py:46]: Assert the contents of `response_msg["data"]` as well

File: app/tests/v1/test_question.py [hunk 3]
```diff
@@ -1,39 +1,32 @@
...
             headers=self.get_authentication_headers(access_token),
             data=json.dumps(self.question)
         )
+        response_msg = json.loads(res.data.decode("UTF-8"))
         self.assertEqual(res.status_code, 201)
+        self.assertTrue(response_msg['data'])
+        self.assertEqual(
+            MEETUPS[0]['questions'][0]['question_id'],
+            response_msg['data'][0]['questions'][0]['question_id']
+        )
```
Comment (RuthNjeri) [inline app/tests/v1/test_question.py:26]: Good job asserting the contents of the data returned as well.

File: app/tests/v1/test_meetup.py (hunk 1 above)
Comment (khwilo) [reply app/tests/v1/test_meetup.py:19]: I have now made changes by asserting that the data found in the meetups list is the one displayed by the response message.

File: app/tests/v1/test_meetup.py (hunk 2 above)
Comment (khwilo) [reply app/tests/v1/test_meetup.py:46]: I have now asserted the contents of the response data by fetching the second meetup created.

File: app/tests/v1/test_question.py (hunk 3 above)
Comment (khwilo) [reply app/tests/v1/test_question.py:26]: Thank you.

**Pull request merged**